src/Components/TableManager/TableManager.js

```diff
@@ -86,7 +90,8 @@ TableManager.propTypes = {
   addTable: PropTypes.func,
   tables: PropTypes.array,
   addCurrentTable: PropTypes.func,
-  currentTable: PropTypes.object
```

Comment:
if you use trailing commas you won't have this and the line below as changed in PRs :100: